meinberlin/apps/projects/templates/meinberlin_projects/project_detail.html

```diff
@@ -15,15 +15,8 @@
 
 {% block extra_messages %}
     {{ block.super }}
-
     {% if project.is_draft %}
-        <ul class="messages">
-            <li class="alert alert--warning" role="alert">
-                <div class="container">
-                    {% translate "This project is not published yet." %}
-                </div>
-            </li>
-        </ul>
+        {% include 'meinberlin_contrib/components/alert.html' with alert_message='This project has not yet been published.' alert_type='info' aria_live_value='polite' %}
```

Comment:
aria_live_value is being passed but doesn't seem to be used, do we need it?

Reply:
good catch. I added an if statement to aria-live in the alert.html which will handle the different cases